spec/openapi-1.0.0.yml

```
@@ -0,0 +1,107 @@
+
+    This describes a set of APIs which should be exposed by a system exposing events.
+
+    Such a system is free to include this description and update descriptions to provide relevance to the current context.
+    Paths, arguments and event property names should not be altered.
+  license:
+    name: 'GNU General Public License v3.0'
+    url: 'https://www.gnu.org/licenses/gpl-3.0.html'
+
+security:
+  - BearerAuth: []
+paths:
+  /statistics:
+    patch:
+      operationId: claimStatistics
+      tags:
+        - List
+      description: 'Claim events occured within the system to be used for statistical purposes.'
+      parameters:
+        - schema:
+            type: string
+            format: date-time
+          in: query
+          name: since
+          description: |
+            From which point in time events should be returned. Defined as a datetime in ISO-8601 format.
+            By specifying a value an external system also signals that all preceeding events already have been claimed and can be deleted by the system.
+            If a value is not provided then all event collected by the system should be returned.
+      responses:
+        200:
+          description: 'The list of events that have occurred since the specified point in time.'
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Event'
+        default:
+          description: 'Unspecified error.'
+
+components:
+  securitySchemes:
+    BearerAuth:
+      type: http
+      scheme: bearer
+  parameters:
+    dateTime:
+      name: 'dateTime'
+      in: 'query'
+      description: 'The identifier of the list to return. Use "default" for the default list.'
+      required: true
+      example: 'default'
+      schema:
+        $ref: '#/components/schemas/DateTime'
+
+  schemas:
+    DateTime:
+      description: When the event occoured. Use ISO-8601 format.
+      type: string
+      format: date-time
+      example: "2019-08-14T12:00:00Z"
+    ItemId:
+      description: An id for an item. What consitutes an item is defined by the system.
+      type: string
+    CollectionId:
+      description: An id for a collection of items. What constitues a collection of items is defined by the system.
+      type: string
+    Event:
+      type: object
+      required:
+        - date
+        - event
+      properties:
+        date:
+          $ref: '#/components/schemas/DateTime'
+        clientId:
+          type: string
+          description: The id of the client system on behalf of which the event is performed.
+        guid:
+          type: string
+          description: The globally unique identifier for the user on behalf of who the event is performed.
+        agency:
+          type: integer
+          description: The agency ID matching the users municipality. This is an ISIL number for the library without the "DK-" prefix.
```

```suggestion
          description: The agency id matching the users municipality. This is an ISIL number for the library without the "DK-" prefix.
```

In all other description you use `id` instead of `ID`.